hack/ec-checks.sh

```diff
@@ -58,3 +58,20 @@ function all_tasks_dir {
     copy_all_task_versions "${task/*\//}" $tasks_dir
  done
 }
+
+function stepactions_dir {
+  if [[ ! -d $1 ]]; then
+    mkdir $1
+  fi
+  local d=$1
+
+  while IFS= read -r -d '' f; do
+      found="$(yq eval '.kind == "StepAction"' "${f}")"
+      [[ ${found} != "true" ]] && continue
+      # Include version in the filename:
+      # stepactions/spam/0.1/spam.yaml -> 0.1-spam.yaml
+      dest="${d}/$(printf "${f}" | cut -d/ -f3- | sed 's_/_-_g')"
```

Comment:
If you'd like to use less dependencies
```suggestion
      dest=${f#*/*/}
      dest=${dest/\//-}
```